source/task_scheduler.c

```diff
@@ -103,6 +138,13 @@ void aws_task_scheduler_schedule_future(
     AWS_ASSERT(task);
     AWS_ASSERT(task->fn);
 
+    AWS_LOGF_DEBUG(
```

Comment:
TRACE

Reply:
My current rule of thumb is to use TRACE only for debugging information that's too costly (spammy) to log under normal circumstances.  This doesn't feel like it meets that, so I'd prefer to keep it DEBUG for now.  Same for the others.